presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -152,6 +152,9 @@
     private boolean ignoreStatsCalculatorFailures = true;
     private boolean printStatsForNonJoinQuery;
     private boolean defaultFilterFactorEnabled;
+    // Give a default 10% selectivity coefficient factor to avoid hitting unknown stats in join stats estimates
+    // which could result in syntactic join order. Set it to 0 to disable this feature
+    private double defaultJoinSelectivityCoefficient = 0.1;
```

Comment:
can we please disable it by default?  This factor may be well suited to the TPCH benchmark or some other test workload, but the accuracy of any "default" selectivity value will vary significantly depending on the data and workload.  Introducing an arbitrary default selectivity can cause many regressions.

Reply:
If one wants a specific join order, I would propose introducing constraints ("hints") rather than relying on the way the SQL is written. (How do I get a bushy plan using only the FROM clause ordering?) Having a default join selectivity will allow better join orders to be chosen even if the selectivity is inaccurate. E.g. when joining 3 tables, where 2 are large and one is small (and there are no table stats), the default would allow the optimizer to choose joining the small table first regardless of how the FROM clause is written. I do not know of another optimizer that does not use defaults for selectivities. There will be parameters to allow FROM clause order to be used.